modules/i3s/src/i3s-slpk-loader.ts

```diff
@@ -0,0 +1,22 @@
+import {LoaderOptions, LoaderWithParser} from '@loaders.gl/loader-utils';
+import {parseSlpk} from './lib/parsers/parse-slpk';
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+export type SlpkLoaderOptions = LoaderOptions & {
+  path?: string;
+  mode?: 'http' | 'raw';
+};
+
+export const SlpkLoader: LoaderWithParser = {
+  name: 'I3S (Indexed Scene Layers)',
+  id: 'slpk',
+  module: 'slpk',
```

Comment:
module should be `i3s` - I believe it is used to construct worker paths. Even if you don't have a worker loader let's keep it correct.